packages/walletkit/bin/build-sdk.js

```diff
@@ -4,31 +4,44 @@ const execSync = require('child_process').execSync
 const fs = require('fs')
 const chalk = require('chalk')
 const path = require('path')
+const { config } = require('dotenv')
 
 function execCmd(cmd) {
   console.log('Running ==> ' + chalk.bold.cyan(cmd))
   execSync(cmd, { stdio: 'inherit' })
 }
 
+function getEnvFile(celoEnv) {
+  const monorepoRoot = path.resolve(path.dirname(__dirname), '../..')
+  const filePath = path.resolve(monorepoRoot, `.env.${celoEnv}`)
+  if (fs.existsSync(filePath)) {
+    return filePath
+  } else {
+    throw new Error(`Unable to find env file for '${celoEnv}'`)
+  }
+}
+
 function isProduction(env) {
-  return env.endsWith('production')
+  const envConfig = config({ path: getEnvFile(env) })
+  return envConfig.parsed['TESTNET_PROJECT_NAME'] === 'celo-testnet-production'
```

Comment:
Did you mean to add a config line of TESTNET_PROJECT_NAME to all the mobile .env files? Should that be in this PR?

Reply:
It uses the env files at the root of the monorepo.

Reply:
Oh or does this access the env files at the monorepo root?